package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codeium/react-code-editor",
-  "version": "1.0.0",
+  "version": "1.0.4",
```

Comment:
Why the skip?

Reply:
don't think the ones in between were version controlled. Current version is 1.0.4, will update to 1.0.5 and then merge and publish